server/bin/deploy

```diff
@@ -335,7 +328,7 @@ bash $SELF_DIR/gen-certs $GEN_CERTS_ARGS
 update_keystore
 
 # update server.xml
-$SUDO python $PROJECT_DIR/bin/update-server-xml.py $CONTAINER_CONF_DIR
+$SUDO python $PROJECT_DIR/bin/update-server-xml.py --version $TC_VERSION $CONTAINER_CONF_DIR
```

Comment:
Sorry if this is nit-picky, but can we use `--tomcat-version`?  `--version` is almost always "show me the version of this program" so using it as an actual functional option creates confusing semantics.

Reply:
Done.